### PR TITLE
fix: StackBlitz dependencies & render(animate: boolean) values

### PR DESCRIPTION
**Pull request by nstuyvesant**
### Updates
- StackBlitz examples open correctly for all packages without prompting for installation of missing packages.
- Aligned the render(animate) parameter in components to pre-1.9.0. If parameter was not previously supplied, the default value for animate is now false.
- Bump for packages within semver range.

### CODE REVIEW

**Comment by theiliad** [comment]
@nstuyvesant I'm no longer seeing a left axis 😃 

image

**Comment by theiliad** [comment]
Hi @nstuyvesant, I'm still seeing the glitch (try navigating between stories)

**Review by theiliad — CHANGES_REQUESTED** [review]

File: packages/core/src/demo/utils/stackblitz/vanilla-js.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  <script src="src/index.js"></script>
+</body>
+</html>`
+
+  const instantiateForGeo =
+`/* Disclaimer: Data only used for demo purposes - not an accurate representation of world map */
+
+async function loadTopoData() {
+  try {
+    const response = await fetch('https://fonts.googleapis.com/css?family=IBM+Plex+Sans+Condensed:300,400|IBM+Plex+Sans:400,600&display=swap')
```
[the remark this answers is not in the document]
Comment (nstuyvesant) [reply packages/core/src/demo/utils/stackblitz/vanilla-js.ts:50]: Clipboard paste issue. Will fix this right away.

[the remark this answers is not in the document]
Comment (nstuyvesant) [reply packages/core/src/demo/utils/stackblitz/vanilla-js.ts]: Pushed correct URL

**Comment by nstuyvesant** [comment]
> Hi @nstuyvesant, I'm still seeing the glitch (try navigating between stories)

@theiliad - having trouble locating the issue with the graphs going from small to normal size. Looking over differences from 1.8.0, one at a time. As this PR fixes the StackBlitz issues and reverts a few of the render(animate) lines, perhaps merge this (assuming everything else looks OK) and do a separate PR for the sizing issue. Alternatively, you may be able to spot the issue quicker than me and correct it in this PR since you have access to my fork.

**Review by theiliad — APPROVED** [review]
LGTM

**Pull request merged**